e2e/pages/adf/loginPage.ts

```diff
@@ -173,7 +173,10 @@ export class LoginPage {
 
     loginToContentServices(username, password) {
         this.settingsPage.setProviderEcm();
-        this.waitForElements();
+
+        browser.controlFlow().execute(async () => {
```

Comment:
I don't think it's necessary to add this method in controlFlow. I suggest to remove the part with deferred and fulfill from waitForElements() , leave just Util.waitUntilElementIsVisible(this.txtUsername) and Util.waitUntilElementIsVisible(this.txtPassword) in there, and then it will not be necessary to add this method in controlFlow.